src/GlobalScreen/Scope/Layout/Builder/StandardPageBuilder.php

```diff
@@ -60,6 +65,14 @@ public function build(PagePartProvider $parts): Page
         $title = $parts->getTitle();
         $short_title = $parts->getShortTitle();
         $view_title = $parts->getViewTitle();
+        $toast_container = $this->ui->factory()->toast()->container();
+
+        $transformer = new ToastsOfNotifications($this->ui->factory(), new ilSetting('notifications'));
```

Comment:
IMO notifications should be collected the same was as other types of notifications are collected (e.g. see SystemInfos, which has an own Collector and is called here src/GlobalScreen/Scope/Layout/Provider/PagePart/StandardPagePartProvider.php:225). we already have the whole `MainNotificationCollector` thing to collect notifications, would be great we could use this pattern for other types of notifications as well.